src/address.js

```diff
@@ -1,14 +1,16 @@
 var Buffer = require('safe-buffer').Buffer
 var bech32 = require('bech32')
-var bs58check = require('bs58check')
+var bbs58check = require('./bs58check')
```

Comment:
Why `bbs` instead of `bs`? What does it mean?

Reply:
I dunno. It is a convention already in place to put a "b" when using local modules